inc/Post_Thumbnails/Component.php

```diff
@@ -40,4 +42,12 @@ public function initialize() {
 	public function action_add_post_thumbnail_support() {
 		add_theme_support( 'post-thumbnails' );
 	}
+
+	/**
+	 * Adds custom image sizes.
+	 */
+	public function action_add_image_sizes() {
+		add_image_size( 'wp-rig-post-thumbnail', 720, 308, true );
```

Comment:
Let's use the built in post thumbnail function, which we can set the size with `set_post_thumbnail_size()`.

And then use `'post-thumbnail'` as the size.

Reply:
So, after discussion, we only need one new image size: ""wp-rig-featured".